java/Glacier2/greeter/build.gradle.kts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) ZeroC, Inc.
+
+plugins {
+    // Register the Slice Tools for Ice and Java plugin without applying it here.
+    // The plugin will be applied to all subprojects in the `subprojects` block below.
+    id("com.zeroc.ice.slice-tools") version "3.8.0-+" apply false
```

Comment:
Just a note, the other demos in this folder still talk about the `Ice Builder plugin` when they should be talking about the `Slice Tools for Ice and Java plugin` like this demo does.